tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think it would be cleaner for ts to output files to a folder, maybe `spec/built`, so that if we add more ts files later we don't need to special case every one in the .ignore files.